README.md

```diff
@@ -45,91 +61,40 @@ Below is a sample playbook:
-force_reactivation | Boolean | Force re-activation even though the Deep Security Agent has already been activated. | false
-
-
-#### Task : manage.yml
-###### action:
-
-* check-in-with-manager
-* create-diagnostic-package
-* create-integrity-baseline
-* run-recommendation-scans
-* scan-for-integrity-changes
-* scan-for-malware
-
-The <code>manage</code> task lets you operate the Deep Security Agent without requiring you to log in to the Deep Security Manager. All of the available actions assume that the Deep Security Agent is installed and activated properly. Most of the actions can be triggered without any additional variables. Only <code>set-policy</code> and <code>set-policyid</code> need a corresponding policy name or id.
-
-###### action: set-policy-by-name
-
-Key | Type | Description | Default
-----|------|-------------|--------
-policy_name | String | The name of the Deep Security policy | 
-
-###### action: set-policy-by-id
-
-Key | Type | Description | Default
-----|------|-------------|--------
-policy_id | Int | The ID of the Deep Security policy | 
-
-```yaml
-- hosts: all
-  roles:
-    - role: deep-security.deep-security-agent
-      action: check-in-with-manager | create-diagnostic-package | ...
-      
-- name: "Set policy for webservers"
-  hosts: web
-  roles:
-    - role: deep-security.deep-security-agent
-      action: set-policy-by-name
-      policy_name: MyCompany - Webserver            
-```
+## Actions Definition
+#### Definition for possible actions to be performed in this role
+Action | Description | Required variables
+------------|-------------|-------------------
+deploy | The <code>deploy</code> task includes the <code>install</code> and <code>activate</code> playbooks internally. | dsm_agent_download_hostname <br /> dsm_agent_download_port <br /> dsm_agent_activation_hostname	 <br/> dsm_agent_activation_port <br/> policy_id <br/> group_id <br/> force_reactivation <br/><br/> **For Multi-Tenancy:** <br/> tenant_id <br /> token/tenant_password
+install | The install task downloads and installs the Deep Security Agent. The installation is skipped if an agent of the same version is already installed. If a newer version of Deep Security Installer is already installed, then the version is upgraded. | dsm_agent_download_hostname <br/> dsm_agent_download_port
+activate | The activate task activates the Deep Security Agent by registering it in Trend Micro Deep Security Manager. By default, activation is skipped if the agent is already activated, unless the force_reactivation attribute is set to true. | dsm_agent_activation_hostname <br/> dsm_agent_activation_port <br/> policy_id <br/> group_id  <br/> force_reactivation <br/><br/> **For Multi-Tenancy:** <br/> tenant_id <br/> token/tenant_password
+set-policy-by-name | Change agent's policy via policy name | policy_id
+set-policy-by-id | Change agent's policy via policy id | policy_name
+check-in-with-manager | Ask the Agent to contact the Manager now. | ~
```

Comment:
For the actions without variable, I'll suggest to use "N/A" in the sample field.